### PR TITLE
award-badges

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Description**

This PR
- awards badges to members' profiles

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c5f6cb61e3305b383c16a7066fa933cd466ef71f at: https://64945b97435cf239a32a301a--layer5.netlify.app

**Comment by vishalvivekm** [comment]
## Congratulations 🎊  @Avi-88 @harkiratsm @Chadha93 @UsmanDev09 ! 🎈

Please take a look at the badges on your [profiles](https://64945b97435cf239a32a301a--layer5.netlify.app/).
Thanks for your contributions to the Layer5 community!

File: src/collections/members/anirudh-jain/index.mdx [hunk 1]
```diff
@@ -11,6 +11,6 @@ bio: I am a software engineer diversing on all part of the stack, eager to learn
 badges:
   - meshery
 blurb:      Mesh Head
-status:     Inactive
+status:     Active
```
Comment (leecalcote) [inline src/collections/members/anirudh-jain/index.mdx:14]: I _think_ that @anirudhjain75 is back, but not sure...

File: src/collections/members/gaurav-chadha/index.mdx [hunk 2]
```diff
@@ -9,6 +9,7 @@ badges:
  - landscape
  - meshery
  - community
+ - meshmap
```
Comment (leecalcote) [inline src/collections/members/gaurav-chadha/index.mdx:12]: Collecting all the badges, @Chadha93 👏

File: src/collections/members/Avishkar/index.mdx [hunk 3]
```diff
@@ -7,6 +7,8 @@ twitter: _4vii_
 linkedin: https://www.linkedin.com/in/avishkargunjal/
 location: Pune, India
 bio: Hi, I am a developer proficient in MERN stack and React native. I am a CSE Undergrad at Vishwakarma Institute of Information Technology , Pune. Currently exploring opensource and learning about devops and cloud native. Would love to collaborate on building something awesome!
+badges:
+- meshery-catalog
```
Comment (leecalcote) [inline src/collections/members/Avishkar/index.mdx:11]: @Avi-88 whoo-hoo!

File: src/collections/members/aadhitya-amarendiran/index.mdx [hunk 4]
```diff
@@ -10,6 +10,7 @@ bio: A student and a guy who loves to learn and develop some stuff. I'm a type w
 blurb: Mesh Head
 badges:
 - meshery
+- docs
```
Comment (leecalcote) [inline src/collections/members/aadhitya-amarendiran/index.mdx:13]: @alphaX86 a solidly earned badge, sir.

File: src/collections/members/harkirat/index.mdx [hunk 5]
```diff
@@ -9,6 +9,9 @@ location: Punjab, India
 bio: "Harkirat Singh is pursuing his Bachelor's in Computer Science From Chitkara University .
 He is a FullStack Web Developer & is passionate about Open Source & DevOps. 
 He is Open to collaborate on Projects and Believes in Learning By Doing."
+badges:
+- meshery
+- meshmap
```
Comment (leecalcote) [inline src/collections/members/harkirat/index.mdx:14]: Who's this @harkiratsm character? 😉 Looking good, Harkirat!!

File: src/collections/members/usman-siddique/index.mdx [hunk 6]
```diff
@@ -7,5 +7,7 @@ linkedin: muhammadusmansiddique
 status: Active
 location: Lahore, Pakistan
 bio: Hello, I am a software developer who specializes in MERN stack development. I am passionate about technology and its impact on the world. My love of coding began during my student years, and I quickly became involved in the tech community, eager to learn and share my skills.Throughout my career, I have remained committed to working with the tech community, and I am particularly enthusiastic about open source software. As a result, I joined the Layer 5 community and continue to thrive here, collaborating with other developers and contributing to open source projects.
+badges:
+- meshmap
```
Comment (leecalcote) [inline src/collections/members/usman-siddique/index.mdx:11]: With your "permission", I'll say that you're coming in strong, @UsmanDev09. 😉

**Comment by leecalcote** [comment]
Merge of this PR will await approval from each of the contributors.

**Review by Chadha93 — APPROVED** [review]

**Review by UsmanDev09 — APPROVED** [review]

**Review by Avi-88 — APPROVED** [review]
Lessgoo!! , thanks guys

**Review by harkiratsm — APPROVED** [review]

**Review by alphaX86 — APPROVED** [review]

**Review by Chadha93 — APPROVED** [review]
Approved by everyone. Merging ✔️

**Pull request merged**